README.md

```diff
@@ -0,0 +1,576 @@
+# sanctuary-type-classes
+
+The [Fantasy Land Specification][FL] "specifies interoperability of common
+algebraic structures" by defining a number of type classes and specifying
+methods which types must provide if they are to be considered members of
+these type classes. To be considered a [Functor][], for example, a type
+must provide a `fantasy-land/map` method which obeys the stated laws.
+
+This project provides four things:
+
+  - `TypeClass`, a function for defining arbitrary type classes;
+  - one `TypeClass` value for each Fantasy Land type class;
+  - lawful Fantasy Land methods for JavaScript's built-in types; and
+  - one function for each Fantasy Land method.
```

Comment:
`toString`, `sequence`, `filter` and `filterM` don't fit in this 4 items. Maybe add a point saying that we also provide some derived methods and helpers?